lib/metanorma/collection_construct_model.rb

```diff
@@ -0,0 +1,215 @@
+module Metanorma
+  class Collection
+    class << self
```

Comment:
@kwkwan Ruby convention: the class name *should* match the filename. Either you call this `Metanorma::CollectionConstructModel` or fix the filename?

Reply:
Fixed in 52489282a66dab35873638ad56961cc51bfda851